web/public/add_game.php

```diff
@@ -1,10 +1,9 @@
 <?php
 declare(strict_types=1);
+global $ELOA, $ELOB;
```

Comment:
generally avoid global. maybe we can talk about it how it can be avoided until the end of the week.

Reply:
Done.